common/transport/coap/src/main/proto/proto_device_info.proto

```diff
@@ -0,0 +1,197 @@
+/**
+ * Copyright © 2016-2023 The Thingsboard Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+syntax = "proto3";
+
+option java_package = "org.thingsboard.server.gen.transport.coap";
```

Comment:
Are we using new protos only for device communication, or do we push some to the queue msgs? I wondering if I should update Kafka UI proto decoders or not. TY

Reply:
only for device communication (they will be parsed and saved like telemetry or attribute)

Reply:
@dashevchenko , let's rename the files to have "efento" in the file name or maybe a sub-folder named "efento".

Reply:
ok

Reply:
The renaming proto folder is not good. Please recheck the suggestion above. Rename files. Or make a subfolder.

`/proto/efento/your_fancy.proto`

You can have another protos in the future and proto generators are also can be set exactly for `proto` folder